### PR TITLE
uploading career countdown report

### PR DESCRIPTION
**Pull request by EllieNodder**
### Trello card

https://trello.com/c/2CSvaBkQ/3445-publish-research-report

### Context

Uploading career countdown report to the /research page on GIT.

### CODE REVIEW

File: app/views/content/research.md [hunk 1]
```diff
@@ -122,6 +130,13 @@ keywords:
   - Research
 ---
 
+## Career Countdown: exploring how students and recent graduates are feeling about taking their next steps
+
+Kindred Agency commissioned research company Censuswide to carry out a survey on behalf of the Get Into Teaching campaign exploring the views of final year university students and recent graduates (from the last three years) in
```
Comment (gemmadallmandfe) [inline app/views/content/research.md:135]: @EllieNodder Richard provided a slightly shorter summary paragraph on the ticket - can we use this instead?

Kindred Agency commissioned research company Censuswide to carry out a survey on behalf of Get Into Teaching exploring the views of final year university students and recent graduates. The purpose was to gain insights into how this cohort feel when it comes to making decisions about their future career and what factors are influencing them.

Comment (EllieNodder) [reply app/views/content/research.md:135]: oops yeah, missed that, will update now

File: app/views/content/research.md [hunk 2]
```diff
@@ -5,6 +5,14 @@ description: |-
 date: "2021-05-17"
```
Comment (GDW7) [inline app/views/content/research.md:5]: @EllieNodder do you know what this date field does (and why it's showing date it does)? Aside from that all looks good.

Comment (EllieNodder) [reply app/views/content/research.md:5]: I think this is the date the research page was first published

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

This should not be merged today but can be merged first thing tomorrow @EllieNodder

**Pull request merged**